mne/io/eyelink/eyelink.py

```diff
@@ -209,14 +203,14 @@ def __init__(
         self.fname = Path(fname)
         self._sample_lines = None  # sample lines from file
         self._event_lines = None  # event messages from file
-        self._system_lines = None  # unparsed lines of system messages from file
         self._tracking_mode = None  # assigned in self._infer_col_names
         self._meas_date = None
         self._rec_info = None
+        self._ascii_sfreq = None
         if gap_desc is None:
             gap_desc = "BAD_ACQ_SKIP"
         else:
-            logger.warn(
+            logger.warning(
```

Comment:
Hmm okay. I incorrectly assumed that I should use mne's logger. I changed it form `warn` to `warning` because `logging.warn` is deprecated. 

I can make a commit to use `mne.utils.warn`.

Reply:
using `mne.utils.warn` will always use the correct logger, that's part of its convenience